boards/artix7_100t.json

```diff
@@ -1,31 +1,41 @@
 {
   "build": {
     "extra_flags": "-D ARTIX7_100T",
-    "f_cpu": "1500000000L",
+    "f_cpu": "50000000L",
     "hwids": [
       [
-        "0x0403",
-        "0x6010"
+         "0x002a",
+         "0x15ba"
       ]
     ],
     "ldscript": "link.ld",
     "mabi": "lp64",
     "march": "rv64imac",
     "mcmodel": "medany",
-    "mcu": "e51"
+    "mcu": "C-Class"
   },
-  "debug": {
-    "onboard_tools": [
-      "ftdi"
-    ]
+   "debug": {
+    "tools": {
+      "shaktidbg": {
+        "server": {
+          "arguments": [
+            "-f", "~/.platformio/packages/framework-shakti-sdk/bsp/third_party/artix7_100t/ftdi.cfg"
+          ],
+          "executable": "~/.platformio/packages/tool-openocd/openocd",
+	  "package": "tool-openocd"
+        }
+
+      }
+    }
   },
   "frameworks": [
     "shakti-sdk"
   ],
   "name": "Arty A7-100: Artix-7 FPGA Development Board",
   "upload": {
-    "maximum_ram_size": 268435456,
-    "maximum_size": 16777216,
+"maximum_size":0,
+      "maximum_ram_size": 134217728,
+    "speed" : 19200,
```

Comment:
What exactly does speed field represent here? Will it be used in the upload routine? Also, the formatting is broken.

Reply:
fixed formatting , we are currently testing the upload command as there is some confusion from the hardware side.

Reply:
For 35t we don't have RAM(DDR integrated) its written to tcm as part of the fpga configuration. For 100t we have DDR as a slandered.